presidio-image-redactor/Pipfile

```diff
@@ -9,6 +9,9 @@ flask = "==1.1.2"
 pytesseract = "==0.3.7"
 presidio-analyzer = ">=1.9.0"
 pillow = "==8.1.1"
+importlib-metadata = { version = "==3.7.3", markers = "python_version<'3.8'" }
+typing-extensions = { version = "==3.7.4.3", markers = "python_version<'3.8'" }
+zipp = { version = "==3.4.1", markers = "python_version>='3.6'" }
```

Comment:
So why the marker in zipp? and do we need it in 3.8?

Reply:
yes, otherwise we get the error:
```
Traceback (most recent call last):
2021-03-22T13:00:59.4701919Z   File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/runpy.py", line 183, in _run_module_as_main
2021-03-22T13:00:59.4702551Z     mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
2021-03-22T13:00:59.4703158Z   File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/runpy.py", line 142, in _get_module_details
2021-03-22T13:00:59.4703746Z     return _get_module_details(pkg_main_name, error)
2021-03-22T13:00:59.4704327Z   File "/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/runpy.py", line 109, in _get_module_details
2021-03-22T13:00:59.4704853Z     __import__(pkg_name)
2021-03-22T13:00:59.4705752Z   File "/home/vsts/.local/share/virtualenvs/presidio-image-redactor-UNwSIaum/lib/python3.7/site-packages/spacy/__init__.py", line 10, in <module>
2021-03-22T13:00:59.4706469Z     from thinc.neural.util import prefer_gpu, require_gpu
2021-03-22T13:00:59.4707760Z   File "/home/vsts/.local/share/virtualenvs/presidio-image-redactor-UNwSIaum/lib/python3.7/site-packages/thinc/__init__.py", line 8, in <module>
2021-03-22T13:00:59.4708457Z     from ._registry import registry
2021-03-22T13:00:59.4709503Z   File "/home/vsts/.local/share/virtualenvs/presidio-image-redactor-UNwSIaum/lib/python3.7/site-packages/thinc/_registry.py", line 1, in <module>
2021-03-22T13:00:59.4710181Z     import catalogue
2021-03-22T13:00:59.4711092Z   File "/home/vsts/.local/share/virtualenvs/presidio-image-redactor-UNwSIaum/lib/python3.7/site-packages/catalogue.py", line 10, in <module>
2021-03-22T13:00:59.4711773Z     import importlib_metadata
2021-03-22T13:00:59.4712823Z   File "/home/vsts/.local/share/virtualenvs/presidio-image-redactor-UNwSIaum/lib/python3.7/site-packages/importlib_metadata/__init__.py", line 6, in <module>
2021-03-22T13:00:59.4713472Z     import zipp
2021-03-22T13:00:59.4714115Z ModuleNotFoundError: No module named 'zipp'
2021-03-22T13:00:59.4944017Z ##[error]Bash exited with code '1'.
2021-03-22T13:00:59.4958854Z ##[section]Finishing: Install deps
```

Reply:
I meant, if we install on >=3.6, why do we need the marker?
And you said that the lib is included in 3.8, so do we need to install zipp in 3.8 as well?

Reply:
oh ok. removing it.
The importlib metadata is included under a different name.